### DISCUSSION TITLE
integration of schemathesis in tests (property based tests)

### DISCUSSION THREAD

**Discussion by devkral** [dymmond/esmerald#310] [General]
given that esmerald outputs an openapi schema it would be nice to use schemathesis to test against.

It would double serve as test as well as documentation how to integrate schemathis

**Comment by tarsil** [comment]
Not 100# familiar with that to be honest so it would be good to see an example or a PR with that to understand?

**Comment by devkral** [comment]
actually such thing is now implemented in edgy. Schemathesis is like a sledge-hammer in contrast to the ModelFactory solution.
Stubbing works quite well now.
